.github/workflows/test-programs.yml

```
@@ -49,13 +57,13 @@ jobs:
         uses: actions-rs/cargo@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)